inc/azure_c_shared_utility/crt_abstractions.h

```diff
@@ -95,6 +97,7 @@ long double strtold_s(const char* nptr, char** endPtr);
 MOCKABLE_FUNCTION(, int, mallocAndStrcpy_s, char**, destination, const char*, source);
 MOCKABLE_FUNCTION(, int, unsignedIntToString, char*, destination, size_t, destinationSize, unsigned int, value);
 MOCKABLE_FUNCTION(, int, size_tToString, char*, destination, size_t, destinationSize, size_t, value);
+MOCKABLE_FUNCTION(, int, uint64_tToString, char*, destination, size_t, destinationSize, uint64_t, value);
```

Comment:
What if the platform does not support uint64_t? 
Maybe it would be safer to define a uint64_t as size_t if it is not defined.

Reply:
However, we use uint64_t already throughout the C SDK. So I think it would not be an issue.